### ISSUE TITLE
dependabot

### ISSUE THREAD

**Issue by diomed** [chinchang/web-maker#345]
what you need is [dependabot](https://dependabot.com) for upping those libraries.
it's free for open source projects.

**Comment by chinchang** [comment]
Are you recommending dependabot for the popular library list? I think it works only for dependencies listed in package.json, not ver sure though.

**Comment by diomed** [comment]
yes, if there's a way to automate process of renewing those, I don't see why not.
